build.gradle

```diff
@@ -194,7 +194,7 @@ void createTestTask(String name, String desc, String environments, int javaVersi
 	boolean releaseDocs = modifiers.contains(Modifiers.GEN_RELEASE_DOCS)
 	boolean docs = modifiers.contains(Modifiers.GEN_NIGHTLY_DOCS) || releaseDocs
 	def artifact = 'build' + File.separator + 'libs' + File.separator
-	if (junit) {
+	if (junit || modifiers.contains(Modifiers.DEV_MODE)) {
```

Comment:
DEV_MODE can have its own variable, why not :)

Reply:
It was only used once before.

Reply:
I know :) it's an optional suggestion